client/src/pages/Profile/ProfileSkills.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import PythonImg from "./img/python.png";
+import JsImg from "./img/js.png";
+import CppImg from "./img/cpp.png";
+import RubyImg from "./img/ruby.png";
+import JavaImg from "./img/java.png";
+import PhpImg from "./img/php.png";
+import GoImg from "./img/go.png";
+import CsharpImg from "./img/csharp.png";
```

Comment:
Any reason these are bitmap and not svg icons?

Reply:
Updated. Replaced png with svg icons